### PR TITLE
DEX-1134 Try to fix DexClientFaultToleranceTestSuite

### PR DESCRIPTION
**Pull request by alexander-branevskiy**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -147,6 +148,7 @@ object Dependencies {
   private val sttpClient = sttpClientModule("core")
   private val sttpPlayJson = sttpClientModule("play-json")
   private val sttpAsyncHttpClient = sttpClientModule("async-http-client-backend-future")
+  private val asyncHttpClient = "org.asynchttpclient" % "async-http-client" % Version.asyncHttpClient
```
Comment (vsuharnikov) [inline project/Dependencies.scala:151]: Why do we need this? This dependency retrieved transitively through `async-http-client-backend-future`, so we need to update `sttpClient` version, e.g. to 3.2.3: https://github.com/softwaremill/sttp/blob/v3.2.3/build.sbt

File: project/Dependencies.scala [hunk 2]
```diff
@@ -46,6 +46,7 @@ object Dependencies {
 
     val sttp = "1.7.2"
```
Comment (vsuharnikov) [inline project/Dependencies.scala:47]: Also please remove this, it is not used anymore )

**Review by vsuharnikov — CHANGES_REQUESTED** [review]

File: project/Dependencies.scala (hunk 2 above)
Comment (alexander-branevskiy) [reply project/Dependencies.scala]: resolved

File: project/Dependencies.scala (hunk 1 above)
Comment (alexander-branevskiy) [reply project/Dependencies.scala]: resolved

**Pull request merged**